File: 5/solution.py
import math

def isPrime(number):
	for i in range(2, number):
		if number % i == 0:
			return False
	return True

primes = [ i for i in range(2, 21) if isPrime(i) ]
lowest_common_multiple = 1

for prime in primes:
	occurrences = int(math.floor(math.log(20, prime)))
	lowest_common_multiple = lowest_common_multiple * (prime ** occurrences)

print lowest_common_multiple

File: 5/solution.rb
class Fixnum
	def is_prime?
		(2...self).each do |i|
			if self % i == 0
				return false
			end
		end
		true
	end
end

primes = []
(2..20).each do |i|
	primes.push i if i.is_prime?
end

lowest_common_multiple = 1

primes.each do |prime|
	occurrences = Math.log(20, prime).floor
	lowest_common_multiple = lowest_common_multiple * (prime ** occurrences)
end

puts lowest_common_multiple